data/data_manager.py

```diff
@@ -237,7 +237,7 @@ def save_data_and_kpisjson(self, fmu_path):
         self.z_fmu.close()
         
     def get_data(self, horizon=24*3600, interval=None, index=None, 
-                 category=None, plot=False):
+                 variables=None, category=None, plot=False):
```

Comment:
Just a note that this may come in handy later too to make forecast data retrieval more efficient.